x/claims/types/params.go

```diff
@@ -25,6 +25,7 @@ var (
 	DefaultAuthorizedChannels = []string{
 		"channel-0", // Osmosis
 		"channel-3", // Cosmos Hub
+		"channel-2", // Injective
```

Comment:
Let's only add this for the tests, otherwise we'd need to support Injective on the dashboard

Reply:
@fedekunze can do, that would mean, that we  don't allow claiming the ibc with `OnRecvPacket` from an EVM chain at launch, correct?